apps/cowswap-frontend/src/modules/swap/services/safeBundleFlow/safeBundleApprovalFlow.ts

```diff
@@ -54,6 +57,13 @@ export async function safeBundleApprovalFlow(
       amountToApprove: context.trade.inputAmount,
     })
 
+    // make sure no pre-approval hooks are included, just as a safety measure
+    if (appDataContainsHooks(orderParams.appData.fullAppData)) {
+      reportAppDataWithHooks(orderParams)
+      // wipe out the hooks
+      orderParams.appData = await updateHooksOnAppData(orderParams.appData, undefined)
+    }
+
```

Comment:
If it's still quoted, as a last resort, remove if from the order params before it's signed.

Reply:
same comment as above